presto-docs/src/main/sphinx/release/release-0.255.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.255
+=============
+
+**Highlights**
```

Comment:
check with committers to see if there is anything worth highlighting. If not please remove this section.

Reply:
@highker @tdcmeehan anything we want to highlight?